payment/app.py

```diff
@@ -1,9 +1,11 @@
 import os
 import atexit
 
-from flask import Flask
+from flask import Flask, Response
 import redis
-
+import requests
```

Comment:
Make sure that `requests` is added to the requirements.txt file in the payment microservice folder, otherwise the Docker container will fail when operating